libbeat/common/kubernetes/metadata.go

```diff
@@ -52,6 +52,15 @@ type MetaGeneratorConfig struct {
 
```

Comment:
Not related, but could you reorder the fields in `MetaGeneratorConfig`? I think that the comment about undocummented fields only refers to `include_creator_metadata`.
```
type MetaGeneratorConfig struct {
	IncludeLabels      []string `config:"include_labels"`
	ExcludeLabels      []string `config:"exclude_labels"`
	IncludeAnnotations []string `config:"include_annotations"`

	LabelsDedot            bool `config:"labels.dedot"`
	AnnotationsDedot       bool `config:"annotations.dedot"`

	// Undocumented settings, to be deprecated in favor of `drop_fields` processor:
	IncludeCreatorMetadata bool `config:"include_creator_metadata"`
}
```